Cargo.toml

```diff
@@ -2,4 +2,11 @@
 
 name = "par-vec"
 version = "0.0.1"
-authors = ["Alexis <a.beingessner@gmail.com>"]
+authors = [
+	"Alexis <a.beingessner@gmail.com>", 
+	"Austin <austin.bonander@gmail.com>"
+]
+
+[dev-dependencies]
+rand = "*"
+threadpool = "*"
```

Comment:
needs at very least the license field; check out the other repos for what we're including in here.